### ISSUE TITLE
howto change default plotting parameters?

### ISSUE THREAD

**Issue by tiyo** [MrPig91/PSChiaPlotter#78]
can i change them?

**Comment by Kar7upeli5** [comment]
I have same question. Perfectly if can use dropdown menu for plot sizes

**Comment by MrPig91** [comment]
Which default parameters are you referring to? I do plan to be able to change plot size in the future, but needed to get the nuts and bolts of the app working before I did that. The only parameter you currently cannot change is the K-Size.

**Comment by tiyo** [comment]
> Which default parameters are you referring to? I do plan to be able to change plot size in the future, but needed to get the nuts and bolts of the app working before I did that. The only parameter you currently cannot change is the K-Size.

I mean all the default parameters that can be set, so as to avoid entering each time a new job is created

**Comment by Jaga-Telesin** [comment]
I interpret this to mean "saving job parameters to a job profile".  Basically so people can set them up once, then simply load them later after launching the GUI for less hassle setting up different jobs.

I *think* this was already on your intended roadmap MrPig91.  :)

**Comment by tiyo** [comment]
> I interpret this to mean "saving job parameters to a job profile". Basically so people can set them up once, then simply load them later after launching the GUI for less hassle setting up different jobs.
> 
> I _think_ this was already on your intended roadmap MrPig91. :)

right!

**Comment by MrPig91** [comment]
Was working on this last night, along with quite a bit of other changes (adding 2nd temp parameter, allowing for all KSizes, basic directory input so you can just give it an old fashion file path). Hope to get it done by Saturday and hope I don't break anything in the process.

**Comment by MrPig91** [comment]
This feature has now been added in the release 1.0.32. A lot was added to release so please report any bugs you come across. Thank :)

**Comment by MrPig91** [comment]
Version 1.0.33 should be the version to update to. Version 1.0.32 had a mistake and was not checking freespace.
